### PR TITLE
Implement seed script

### PR DESCRIPTION
**Pull request by chaynsaw**
Implement sequelize
Implement two tables
Normal distribution of amount of people paying average price still pending

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -7,10 +8,29 @@ class App extends Component {
   render(props) {
     return (
       <div>
-        What up.
```
Comment (ivanlee4289) [inline client/components/app.jsx]: um..hi.

File: client/components/app.jsx [hunk 2]
```diff
@@ -7,10 +8,29 @@ class App extends Component {
...
+        "id": id,
+        "first_name": firstName,
+        "last_name": lastName,
+        "email": email
+    });
+  }
+  return { "data": users }
+}
+let dataObj = generateUsers();
+console.log(dataObj);
```
Comment (ivanlee4289) [inline client/components/app.jsx]: very easy to understand function. Good job!

**Comment by chaynsaw** [comment]
@Cokane415 @ivanlee4289 @umairnadeem123 Hello all, please see my pull request. Thanks!

File: server/server.js [hunk 3]
```diff
@@ -1,8 +1,36 @@
 const express = require('express')
```
Comment (ivanlee4289) [inline server/server.js]: I suggest including bodyparser as well

Comment (chaynsaw) [reply server/server.js]: Good catch. Installed.

**Pull request merged**